cmd/tendermint/commands/reindex_event.go

```diff
@@ -192,19 +212,19 @@ func eventReIndex(cmd *cobra.Command, es []indexer.EventSink, bs state.BlockStor
 			if e.NumTxs > 0 {
 				batch = indexer.NewBatch(e.NumTxs)
 
-				for i, tx := range b.Data.Txs {
+				for i := range b.Data.Txs {
 					tr := abcitypes.TxResult{
 						Height: b.Height,
 						Index:  uint32(i),
-						Tx:     tx,
+						Tx:     b.Data.Txs[i],
```

Comment:
Why this change?

Reply:
to avoid hanging on to a stale reference